packages/fx-core/tests/plugins/resource/frontend/unit/generateArmTemplates.test.ts

```diff
@@ -37,49 +37,50 @@ describe("FrontendGenerateArmTemplates", () => {
     const result = await frontendPlugin.generateArmTemplates(pluginContext);
 
     // Assert
-    const testModuleFileName = "frontend_hosting.bicep";
+    const testModuleFileName = "frontendProvision.result.v2.bicep";
     const mockedSolutionDataContext = {
       Plugins: activeResourcePlugins,
       PluginOutput: {
         "fx-resource-frontend-hosting": {
           Modules: {
             frontendHostingProvision: {
-              Path: `./${testModuleFileName}`,
+              ProvisionPath: `./${testModuleFileName}`,
             },
           },
         },
       },
     };
     chai.assert.isTrue(result.isOk());
     if (result.isOk()) {
-      const expectedResult = mockSolutionUpdateArmTemplates(
+      // const expectedResult = mockSolutionUpdateArmTemplates(
```

Comment:
remove the unused comment in all test files?

Reply:
sure